src/svgs/glyphs/ivermectin-glyph.svg

```diff
@@ -0,0 +1,41 @@
+<svg
+  width="48"
+  height="48"
+  viewBox="0 0 48 48"
+  fill="none"
+  xmlns="http://www.w3.org/2000/svg"
+>
+  <title>Ivermectin Ingredient Glyph</title>
+  <g clip-path="url(#clip0)">
```

Comment:
We should be able to remove `<g>` and `<defs>` (and children of `<defs>`) from these files without visual changes, see docs: https://www.notion.so/curology/Working-with-SVGs-c70a743abbe144f1898e9c1095b5b267#ae5c0648f0a2440d8ae87eb4ff2f337c

To test, you can make the changes, and since the visual regressions tests have since been approved (https://www.chromatic.com/build?appId=5f6e46f1e7081000221523cf&number=1300) we should expect to see the rebuild pass as well.